source/administration/legacy-upgrade.md

```diff
@@ -1,17 +1,109 @@
-# Legacy Upgrade Guide
+# Upgrading from an Unsupported Version
 
-This is a legacy upgrade guide for servers running Mattermost version v2.2.0 and earlier. If you are upgrading a server running Mattermost v3.0.x or later, please refer to our most [current upgrade guide](https://docs.mattermost.com/administration/upgrade.html).
+These instructions apply only if you are upgrading from an unsupported version of Mattermost to the latest version. Only the current released version and the two previous versions are supported.
 
-- [Upgrade Team Edition](https://docs.mattermost.com/administration/upgrade.html#upgrade-team-edition)
-  - [Upgrade Team Edition to 3.0.x](https://docs.mattermost.com/administration/legacy-upgrade.html#upgrade-team-edition-to-3-0-x)
-  - [Upgrade Team Edition to 2.2.x and earlier](https://docs.mattermost.com/administration/legacy-upgrade.html#upgrade-team-edition-to-2-2-x-and-earlier)
-- [Upgrade Enterprise Edition](https://docs.mattermost.com/administration/upgrade.html#upgrade-enterprise-edition)
-  - [Upgrade Enterprise Edition to 3.0.x](https://docs.mattermost.com/administration/legacy-upgrade.html#upgrade-enterprise-edition-to-3-0-x)
-  - [Upgrade Enterprise Edition to 2.2.x and earlier](https://docs.mattermost.com/administration/legacy-upgrade.html#upgrade-enterprise-edition-to-2-2-x-and-earlier)
+If you are upgrading a server that is already running a supported version, See [Upgrading Mattermost Server](../../administration/upgrade.html).
 
-## Upgrade Team Edition
+## Upgrade Team Edition from 3.0.0 and later
 
-### Upgrade Team Edition to 3.0.x
+### Important Notices
+
+1. Security related changes were made in 3.9.0 that cause any previously created team invite links, password reset links, and email verification links to no longer work. You must update any place where you have published these links.
```

Comment:
do we want to clarify these notes apply to all newer versions too? I.e. if you upgrade from 3.6 to 3.10, all of these apply.